bendigobank.rb

```diff
@@ -2,69 +2,79 @@
 
 require 'watir'
 require 'nokogiri'
+require 'selenium-webdriver'
+require 'json'
 require 'pry'
 require_relative 'account'
 require_relative 'transaction'
 
-# Operate over a bank account.
+# class operates over a bank account.
 
 class Bendigobank
+
+
   def execute
-    login
-    fetch_accounts
-    fetch_transactions
-    show_result
+   connect
+   fetch_accounts
+   fetch_transactions
+    puts @accounts
+    close
   end
 
+  binding.pry
+  browser = Watir::Browser.new(:chrome)
+  browser.goto('https://demo.bendigobank.com.au')
+
```

Comment:
Move this URL to constant.

Variable browser should instance variable. It will we visible at all methods.
You can initialize this variable in your login method

```ruby
def login
  @browser = Watir::Browser.new(:chrome)
  @browser.goto(BANK_URL)
  # other login actions
  # ... 
end
```

Do not forget to delete `binding.pry` in the final version of code.